Tests/Fixtures/Controller/AnnotatedPrefixedController.php

```diff
@@ -20,7 +21,7 @@
  */
 class AnnotatedPrefixedController extends Controller
 {
-    public function getSomethingAction()
+    public function getSomethingAction(Request $request)
```

Comment:
To check that the routing is working and that this type hints do not affect the routes.